index.bs

```diff
@@ -4102,13 +4102,18 @@ to the dictionary members defined on |D| and on any of |D|’s
 On a given dictionary value, the presence of each dictionary member
 is optional, unless that member is specified as required.
 When specified in the dictionary value, a dictionary member is said to be
-<dfn id="dfn-present" export>present</dfn>, otherwise it is [=present|not present=].
+<dfn id="dfn-present" export lt="present|not present" for="dictionary member">present</dfn>, otherwise it is [=not present=].
```

Comment:
Hmm maybe, but we already have default value for="dictionary member", so I went for consistency.